### DISCUSSION TITLE
db model通用字段抽成一个通用的model

### DISCUSSION THREAD

**Discussion by jinronga** [cloudzenith/DouTok#54] [General]
## Question

像这种通用的字段是不是可以抽成一个通用的model 然后组合进每个具体业务model 中去。
![image](https://github.com/user-attachments/assets/89de606f-c7e6-41f6-9555-fa205ffbb9da)

**Comment by TremblingV5** [comment]
实际上是应该这样做的，但这里其实使用gorm/gen直接生成出来的代码，所以也不需要刻意去更改它

**Comment by jinronga** [comment]
这些是根据表结构生成出来的代码吗

**Comment by BaiZe1998** [comment]
是的

**Comment by TremblingV5** [comment]
> 这些是根据表结构生成出来的代码吗

生成工具所在仓库：https://github.com/cloudzenith/gen/tree/master/tools/gentool

后端项目目录下有gen.yml来控制具体的生成参数，通过Makefile的`gen-db`命令来生成

**Comment by jinronga** [comment]
大佬，优雅啊
